source/exercises/mailroom-part1.rst

```diff
@@ -4,14 +4,16 @@
 Mailroom Part 1
 ================
 
-**Using Python's basic data types and logic for a full program**
+**Use Python's basic data types and logic for a full program**
+
+.. This needs some updating now that we've broken it out into multiple files
```

Comment:
can this line be removed plz?

Reply:
or what should be done here exactly?

Reply:
Kris had added text to make it clear that the exercise was in multiple parts, each associated with a different lesson. But now that the pages have been separated, that text needs some updating.